UI/CUIControls.h

```diff
@@ -594,4 +595,58 @@ class RotatingGraphic : public GG::StaticGraphic {
     float   m_phase_offset;
 };
 
+class CUIToggleButton;
```

Comment:
The `virtual void Render{Un,}Checked(const CUIToggleButton& button) const;` don't need a CUIToggleButton as parameter, using a GG::StateButton is sufficient.

If you take a look at the other comment this will become more clear.

> Overwrite the CUIToggleButtonRepresenter::Render(comst GG::StateButton&) method instead.

For reference you can take a look at UI/CUIControls.cpp:416. There I overwrite the `GG::StateButtonRepresenter:Render(comst GG::StateButton&) const` to do the rendering.  You should do the same for the `CUIToggleButtonRepresenter`.